plugins/generator-1.20.1/forge-1.20.1/block.definition.yaml

```diff
@@ -388,6 +388,18 @@ templates:
     writer: json
     name: "@MODASSETSROOT/models/block/@registryname_inventory.json"
 
+block_base_properties:

```

Comment:
Leaves, for example also have properties, that this PR seems to ignore:

![image](https://github.com/user-attachments/assets/2f6ba39a-7972-48c9-8ebb-f846e30da1e4)

Please, again, go carefully through each block base and make sure to cover them all. I can't trust PRs that have bugs on every corner and it takes a lot of time to review such PRs as every line needs to be questioned

Reply:
:heavy_check_mark: Fixed